### DISCUSSION TITLE
Moving from wiki to site or readme

### DISCUSSION THREAD

**Discussion by tomap** [hexojs/hexo#4798] [Ideas]
Hello,

I'm really not in love with GitHub wikis. I would like to suggest to move the content of the wikis to either the readme of each repos or hexo site.

This content is often obsolete, sometimes duplicate of the readme and most of the time not linked.

Tell me what you think!

I already suggested to do it here: https://github.com/hexojs/hexo-renderer-markdown-it/pull/154

**Comment by yoshinorin** [comment]
> I'm really not in love with GitHub wikis. I would like to suggest to move the content of the wikis to either the readme of each repos or hexo site.

Basically, I agree with your opinion.
Exceptionally, I think we should not move the [site list](https://github.com/hexojs/hexo/wiki/Sites) from wiki.

**Reply by tomap** [reply]
Agreed. I'll start doing PRs to move content from one side to the other. Probably leaving the big chunks for later :)
